git_machete/github.py

```diff
@@ -126,8 +126,12 @@ def get_token_from_file_in_home_directory() -> Optional[str]:
 
 
 def __extract_failure_info_from_422(response: Any) -> str:
+    additional_msg: str = ''
     if response['message'] != 'Validation Failed':
```

Comment:
Hmmm it's pretty brittle... what if the failure is NOT due to reviewers being outside the organization? `message` being `Validation Failed` can mean many different things in principle (even if we haven't observed anything else so far)